rudder_analytics/client.py

```diff
@@ -234,20 +233,7 @@ def _enqueue(self, msg):
         # anonymous id need not be forced.
         # msg['anonymousId'] = msg['anonymousId'] or self.anonymoys_id
 
-        # copy the userId to context.traits
-        if 'userId' in msg.keys() :
```

Comment:
Earlier the userId and AnonymousId used to be present in the `context.traits` if they are being passed by the end-user. 

Now since we are removing `context.traits` and having `traits` only on the root both `userId` and `anonymousId` will be present on the root only and not inside the traits.

Can we check if any destination on the transformer side gets affected since the `userId` and `anonymousID` is not present in the traits?

Also by making this are we making Python SDK different from other server-side SDK's?

Reply:
@desusai7 We discussed this internally. Rudder Server handles userId/anonymousId the same way as it does for traits.
Also customers will be notified ahead of release.

Reply:
Okay got it